omnibus/cookbooks/firezone/attributes/default.rb

```diff
@@ -99,13 +99,31 @@
 # Local email/password authentication is enabled by default
 default['firezone']['authentication']['local']['enabled'] = true
 
-# If using the 'okta' authentication method, set 'enabled' to true and configure relevant settings below.
+# OIDC Authentication
+# Any OpenID Connect provider can be used here.
+# Example of a Google setup
+default['firezone']['authentication']['oidc']['google'] = {
+    discovery_document_uri: "https://accounts.google.com/.well-known/openid-configuration",
+    client_id: "CLIENT_ID",
+    client_secret: "CLIENT_SECRET",
+    redirect_uri: "https://example.com/session",
```

Comment:
Would it help to use a more realistic example redirect_uri here? e.g.
 `"https://firezone.example.com/auth/oidc/google/callback"`

Reply:
Fixed in https://github.com/firezone/firezone/pull/573/commits/948b65d4b4fbf760b7f0dd5a57aa10d20c7566fe